code/game/machinery/computer/camera.dm

```diff
@@ -83,6 +83,9 @@
 		ui = new(user, src, ui_key, "CameraConsole", name, 870, 708, master_ui, state)
 		ui.open()
 
+/obj/machinery/computer/security/ui_close(mob/user)
+	watchers -= user.UID()
+
```

Comment:
I would make this call the parent.

We have no other logic for `ui_close()` yet but if we ever apply it to consoles lower in the chain I can see this being one of those things thats a pain to deal with.

Reply:
Can do.